### PR TITLE
Adding uidmap to install steps for ubuntu

### PR DESCRIPTION
**Pull request by vsoch**
Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

This is (another) small PR to add the install of uidmap for Ubuntu users. Without it, the user will see a warning about not having a range of uid/gids:

```bash
$ podman pull docker://fedora:latest
WARN[0000] using rootless single mapping into the namespace. This might break some images. Check /etc/subuid and /etc/subgid for adding subids 
Trying to pull docker://fedora:latest...Getting image source signatures
Copying blob 01eb078129a0 done
Copying config d09302f77c done
Writing manifest to image destination
Storing signatures
ERRO[0010] Error while applying layer: ApplyLayer exit status 1 stdout:  stderr: there might not be enough IDs available in the namespace (requested 192:192 for /run/systemd/netif): lchown /run/systemd/netif: invalid argument 
ERRO[0011] Error pulling image ref //fedora:latest: Error committing the finished image: error adding layer with blob "sha256:01eb078129a0d03c93822037082860a3fefdc15b0313f07c6e1c2168aef5401b": ApplyLayer exit status 1 stdout:  stderr: there might not be enough IDs available in the namespace (requested 192:192 for /run/systemd/netif): lchown /run/systemd/netif: invalid argument 
Failed
(0x189ade0,0xc0007caa20)
```
And the build ultimately fails. It isn't actually the case that there aren't enough ids:

```bash
$ cat /etc/subuid
vanessa:100000:65536
$ cat /etc/subgid
vanessa:100000:65536
```
but that we need to install uidmap (thanks @giuseppe!)  

```bash
sudo apt-get update && sudo apt-get install -y gidmap
```
When the library is installed, it goes away!
```bash
$ podman pull docker://fedora:latest
Trying to pull docker://fedora:latest...Getting image source signatures
Copying blob 01eb078129a0 done
Copying config d09302f77c done
Writing manifest to image destination
Storing signatures
d09302f77cfcc3e867829d80ff47f9e7738ffef69730d54ec44341a9fb1d359b
```
And we have container!

So this isn't an issue with podman, just documentation. The original issue was reported on (a closed similar issue) here https://github.com/containers/libpod/issues/2542#issuecomment-482582611

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -108,6 +110,7 @@ yum install -y \
   ostree-devel \
   pkgconfig \
   runc \
+  uidmap \
```
Comment (TomSweeneyRedHat) [inline install.md]: This should go under the Debian and Ubuntu section not the Fedora/RHeL section no?

Comment (vsoch) [reply install.md]: lol that’s embarrassing :p I’ll fix it up in a few minutes!

Comment (vsoch) [reply install.md]: Fixed.

**Comment by giuseppe** [comment]
Thanks!

/lgtm

**Comment by vsoch** [comment]
```
Summarizing 4 Failures:

[Fail] Podman healthcheck run [It] podman healthcheck that should fail 
/var/tmp/go/src/github.com/containers/libpod/test/e2e/healthcheck_run_test.go:58

[Fail] Podman healthcheck run [It] podman healthcheck that should fail 
/var/tmp/go/src/github.com/containers/libpod/test/e2e/healthcheck_run_test.go:58

[Fail] Podman healthcheck run [It] podman healthcheck that should fail 
/var/tmp/go/src/github.com/containers/libpod/test/e2e/healthcheck_run_test.go:58

[Fail] Podman checkpoint [It] podman checkpoint all running container 
/var/tmp/go/src/github.com/containers/libpod/test/e2e/checkpoint_test.go:210

Ran 612 of 636 Specs in 1357.566 seconds
FAIL! -- 611 Passed | 1 Failed | 1 Flaked | 0 Pending | 24 Skipped


Ginkgo ran 1 suite in 22m49.10521171s
Test Suite Failed
Makefile:184: recipe for target 'ginkgo' failed
make: *** [ginkgo] Error 1
+ exit_handler
+ set +ex
.
The integration test end time at the tone will be:	2019-04-12T20:20:14+00:00
BLEEEEEEEEEEP!
.

Exit status: 2
```

**Comment by mheon** [comment]
Healthchecks is a known flake, restarting

**Pull request merged**